### PR TITLE
Foundations - JavaScript Basics - Add 'Installing Node.js' lesson 

### PR DESCRIPTION
**Pull request by programmurr**
#### Because:

This commit is the second task of [#23248](https://github.com/TheOdinProject/curriculum/issues/23248) on the curriculum repo. The 'Installing Node.js' lesson has been added to the curriculum repo and this PR adds the lesson to the TOP site.

#### This commit

- Adds the Installing Node.js lesson to db/fixtures/lessons/foundation_lessons.rb
- Adds the Installing Node.js lesson to db/fixtures/paths/foundations/seed.rb

If this PR is approved I will then update the JavaScript Exercises [README](https://github.com/TheOdinProject/javascript-exercises/blob/main/README.md) to reflect the change.

#### Checklist
 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/theodinproject/wiki/Contributing-Guide)?
 - [x] You have verified the tests and linters all pass against your changes?
 - [x] You have included automated tests for your changes where applicable?

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -630,4 +630,4 @@ RUBY VERSION
    ruby 2.7.4p191
 
 BUNDLED WITH
-   1.17.3
+   2.1.4
```
Comment (programmurr) [inline Gemfile.lock:633]: I've just noticed this change in bundler version. Will this cause an issue? Must have happened when I ran the `bundle install`. At first it didn't run correctly so I ran it with the `bundle update --bundler` command.

Comment (KevinMulhern) [reply Gemfile.lock:633]: It's all good Conor, we've been meaning to upgrade bundler anyway 😄

**Review by KevinMulhern — COMMENTED** [review]
Sorry for the delay on this @programmurr, thanks for putting this together it all looks great.

I just want to clarify the placement. Do you think this would fit better in the installations section of Foundations or is the placement right before the exercises lesson critical? 

The reason I ask is because the JS section is getting a little bit stacked at the moment and it seems like this lesson could fall under the installation sections remit nicely. 

Hit me up on discord if you'd rather discuss it there mate.

**Comment by programmurr** [comment]
Hi, sorry for my delay in replying  @KevinMulhern  it's been a mental couple of days! 
I reckon it could go to the installations but then they wouldn't use it for quite a while until the actual JavaScript Exercises themselves. With the command line and git lessons, they jump straight into using them. But with Node they would install it and then detour first into HTML, CSS, etc.

I do agree the JS section is a little stacked. This just brings clarity, I think, to an important installation step that at the moment is contained inside another README (the JavaScript Exercises README).

**Review by KevinMulhern — APPROVED** [review]
Thanks for that explanation @programmurr, makes sense to me mate 🚀

**Pull request merged**